### PR TITLE
core/test: fix spelling in Java and CFC files

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

**Comment by michaeloffner** [comment]
@cfmitrah can you please resolve the conflicts

**Comment by BaseMax** [comment]
any updates here?

File: test/components/_Administrator.cfc [hunk 1]
```diff
@@ -1148,7 +1148,7 @@ component extends="org.lucee.cfml.test.LuceeTestCase"{
 				});
 			});
 
-			xdescribe( title="test log functions", body=function() {
+			describe( title="test log functions", body=function() {
```
Comment (zspitzer) [inline test/components/_Administrator.cfc:1151]: this isn't a typo, that's shorthand to disable a test!

**Comment by michaeloffner** [comment]
manually adapt the PR https://github.com/lucee/Lucee/commit/03de72326304c35c32655511c5a255d843c8143a 
thanks @jbampton for your work on this

**Pull request closed without merging**